sdk/healthinsights/azure-healthinsights-radiologyinsights/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.0.1 (Unreleased)
+## 1.1.0 (Unreleased)
+Extending scope with inferences for scoring and assessment, quality measure and clinical guidance
```

Comment:
This sentence can probably go under Features Added. Also, if there are more specific changes like new methods, models, etc added in this version that you can include, that would be helpful for users.